client/my-sites/stats/feedback/style.scss

```
@@ -1,8 +1,60 @@
+@import "@automattic/components/src/styles/typography";
+
 .stats-feedback-card {
 	background: var(--studio-white);
 	border: 1px solid var(--studio-gray-5);
 	border-radius: 5px; // stylelint-disable-line scales/radii
 
 	padding: 20px;
-	margin: 32px;
+
+	@media (max-width: 782px) {
```

```suggestion
	@media (max-width: $break-medium) {
```
Yes, there are a bunch of [breakpoints](https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_breakpoints.scss#L11) we can use.

---

Thanks Dognose. I'll test with the breakpoints in that file. Should work well for this use-case.